src/http/api/resources/files.js

```diff
@@ -245,7 +247,7 @@ exports.add = {
       ipfs.files.addPullStream(options),
       pull.map((file) => {
         return {
-          Name: file.path ? file.path : file.hash,
+          Name: file.path, //addPullStream already turned this into a hash if it wanted to
```

Comment:
This was already being done in addPullStream so I removed it from here, because addPullStream has a new case where it is leaving path as an empty string and wants it that way.